lib/commands/web.js

```diff
@@ -277,45 +283,6 @@ helpers.convertElementsForAtoms = function convertElementsForAtoms (args = []) {
   return resultArgs;
 };
 
-helpers.parseExecuteResponse = function parseExecuteResponse (res) {
```

Comment:
why don't we need this anymore?

Reply:
this helper was actually never called

Reply:
this helper was initially used by https://github.com/appium/appium-ios-driver/blob/9755e309af6f5cbd4a895ca3f157650a943eff5b/lib/commands/web.js#L346, but I find it redundant to have this call in `waitForAtom`, since we only fetch element identifiers while performing elements lookup. That is why the caching functionality only targets `findWebElementOrElements` API now